apps/mobile/src/features/send-refactor/components/asset-picker/asset-picker-item.tsx

```diff
@@ -0,0 +1,43 @@
+import { ReactElement, ReactNode, cloneElement, useRef, useState } from 'react';
+import { View } from 'react-native';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { HEADER_HEIGHT } from '@/shared/constants';
+
+const parentTopPadding = 16;
+
+// Wrapper around asset items for measuring and reporting vertical offsets for animation.
```

Comment:
I'm trying to understand what's happening here. Is this why you also are adding an `onPress` to `BitcoinBalance` ? 

What's the benefit of doing this vs using `forwardRef` if needed?